cli/src/commands/new.rs

```diff
@@ -53,9 +53,9 @@ pub(crate) struct NewArgs {
         add = ArgValueCompleter::new(complete::revset_expression_all),
     )]
     revisions: Option<Vec<RevisionArg>>,
-    /// Ignored (but lets you pass `-o/-d`/`-r` for consistency with other
+    /// Ignored (but lets you pass `-o`/`-d`/`-r` for consistency with other
```

Comment:
I think we should decide on a consistent description instead of having these two:

> Equivalent to positional arguments

> Ignored (but lets you pass `-o`/`-d`/`-r` for consistency with other commands)

I don't know which is better, or if we should have a combination of them. I can see value both in saying that it's equivalent to positional arguments and that it's for consistency. Perhaps something like this:

> Equivalent to positional arguments (lets you pass `-o`/`-d`/`-r` for consistency with other commands)

What do you think?